Bio/SearchIO/_model/query.py

```diff
@@ -761,10 +761,10 @@ def sort(self, key=None, reverse=False, in_place=True):
             self._transfer_attrs(obj)
             return obj
 
-    # Default function for hit_key_function argument in QueryResult.__init__
-    @staticmethod
-    def _hit_key_func(hit):
-        return hit.id
+
+# Default function for hit_key_function argument in QueryResult.__init__
```

Comment:
Can you turn that comment into a function docstring? e.g. ``Default function for hit_key_function argument in QueryResult.__init__ (PRIVATE).`` would be fine.